integration/test/test_hint_time.py

```diff
@@ -153,9 +157,9 @@ def test_hint_time(self):
             expect = 1.0
             actual = raw_totals['time-hint-memory (s)']
             util.assertNear(self, expect, actual, msg=msg)
-            msg = "Application totals should have four seconds of total time"
-            expect = 4.0
-            actual = raw_totals['sync-runtime (s)']
+            msg = "Application totals should have nine seconds of total time"
+            expect = 9.0 + overhead_time
```

Comment:
There should be a comment here explaining why the total time is different than what is setup in the test_hint_time.cpp (4 seconds presently).